image/templates/helm/stackrox-secured-cluster/templates/collector.yaml.htpl

```diff
@@ -100,6 +102,14 @@ spec:
         - mountPath: /run/secrets/stackrox.io/certs/
           name: certs
           readOnly: true
+        [<- if .FeatureFlags.ROX_RUNTIME_FILTERS >]
```

Comment:
Will this appear in the release artifacts?

Reply:
No, our templating engine will remove the lines with `[< ... >]` and either render the inner part or remove it, depending on what the status of the feature flag is.